lib/ohai/application.rb

```diff
@@ -96,6 +102,10 @@ def run_application
     else
       puts ohai.json_pretty_print
     end
+    if Ohai::Config[:profiling]
+      puts ohai.json_pretty_print(
```

Comment:
I'd like to see this go to stderr, in the same way we (well, Dan) changed knife to output statistics to stderr and "real" output to stdout.